scripts/qa_crowbarsetup.sh

```diff
@@ -2337,6 +2337,7 @@ function oncontroller_testsetup()
         echo 1 > /proc/sys/kernel/sysrq
         ./run_tempest.sh -N $tempestoptions 2>&1 | tee tempest.log
         tempestret=${PIPESTATUS[0]}
+        testr last --subunit | subunit-1to2 | tee tempest.subunit.log
```

Comment:
Uhmm  .. to see the tempest result in the Jenkins console? But now I see that two lines above there is a tee to exactly that.